### PR TITLE
Widget list Page

### PR DESCRIPTION
**Pull request by heoegema**
Adding Proof of Concept (Can we render all widgets from the blimp-live-widgets library dynamically?) 

Answer: YES. Turns out we can. 

PreReqs: 
-Link the blimp-live-widgets library 
-To do this follow the [README](https://github.com/blimp-live/blimp-widgets)

- Made a new page 
/widget-list/  (Go check it out) 

image

Future: 
- Will need to wrap each 'widget' in a drag and drop functionality
- WIll probably make this a 'widget list' component instead of a page. (Again, need to have drag and drop and how we're doing that)

### CODE REVIEW

**Comment by heoegema** [comment]
Checks are failing because library isn't published....

**Comment by heoegema** [comment]
Checks are no longer failing because the library is published.

File: src/containers/developer-page/index.tsx [hunk 1]
```diff
@@ -27,11 +27,13 @@ export class DeveloperPage extends React.Component<Props> {
...
               <Link to="/developer/">Developer</Link>
             </li>
+            <li>
+              <Link to="/widget-list/">Widget List</Link>
+            </li>
           </ul>
         </nav>
         <h1>Developer</h1>
         <button onClick={() => this.props.actions.createDashboard('Stanley Huang', 69)}>Hello</button>
-        <button onClick={() => this.props.actions.saveDashboard(69, 'Riley Gowanlock', {})}>Save</button>
```
Comment (BhavikaSharma) [inline src/containers/developer-page/index.tsx]: Just want to make sure we did want to delete this line.

Comment (heoegema) [reply src/containers/developer-page/index.tsx]: Lol no. Oops

File: src/containers/widget-list/index.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+  // This loops through the set of widgets and
+  // renders them
+  // This is just done on a page for now
+  // TODO: In future need containers and to have the Drag and Drop functionality
+  // around this
+  generateListOfWidgets() {
+    const widgetComponents = []
+    for (let widget in widgets) {
+      if (widget != 'ClockThemes') {
```
Comment (BhavikaSharma) [inline src/containers/widget-list/index.tsx]: Also just so we have it documented, what is this check for?

**Pull request merged**